### DISCUSSION TITLE
References on performance and system requirements

### DISCUSSION THREAD

**Discussion by gilgamezh** [tiangolo/nginx-rtmp-docker#81] [Questions] [answered]
Do you have any reference on the system requirements? 

I think this would be a super good backup for some python events but I can't find a good source to check what are the expected system requirements.

**Comment by Balls0fSteel** [comment, marked as answer]
I know this is an old ticket, but this only re-streams source content. So system requirement is very low, but you need the bandwidth. A small VPS can do this work basically.
